HW04/ivan_klimov/spec/mentor_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+require_relative '../lib/person'
+require_relative '../lib/mentor'
+require_relative '../lib/student'
+require_relative '../lib/homework'
+require_relative '../lib/notification'
+
+RSpec.describe Mentor do
+  let(:mentor) { described_class.new(name: 'Olga', surname: 'Urban') }
+  let(:student) { Student.new(name: 'Ked', surname: '500') }
+  let(:homework) { Homework.new(title: 'task1', description: 'Solve 5/6', student: student) }
+
+  describe '#add_homework' do
+    it 'returns homework' do
+      test_homework = mentor.add_homework(title: 'task1', description: 'Solve 5/6', student: student)
+      expect(test_homework).to be_kind_of(Homework)
+    end
+
+    it 'makes notification to the student' do
+      test_homework = mentor.add_homework(title: 'task1', description: 'Solve 5/6', student: student)
+      expect(student.notifications.first.include?(test_homework.title)).to eq true
+    end
+  end
+
+  describe '#subscribe_to!' do
+    it 'add notification to the student about subscribers' do
```

Comment:
try to correct everywhere, please
```suggestion
    it 'adds notification to the student about subscribers' do
```